Valigator.Core/Helpers/ValigatorModelBaseHelpers.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+using Valigator.Core.Helpers;
+
+namespace Valigator.Core.Helpers
+{
+
+	public static class ValigatorModelBaseHelpers
```

Comment:
Methods to create `CustomPropertyInfo` objects which are used to fake property setters.